elyra/templates/kfp_template.jinja2

```diff
@@ -58,13 +64,13 @@ if __name__ == "__main__":
 
     client = kfp.Client(host='{{ api_endpoint }}')
 {% endif %}
-    name = '{{ pipeline_name }}'
-    exp = client.create_experiment('{{ pipeline_name }}',
+    run_name = '{{ pipeline_name }}-run'
```

Comment:
Should the 'run' be associated with the `experiment_name` rather than `pipeline_name`?

Reply:
No, because a run (also sometimes referred to as job) is an instance of a pipeline execution. See my earlier clarification what experiments are in response to your earlier comment.

Reply:
Isn't `run_name` the time-stamped `pipeline_name` now?  Does it require its own parameter since it's not always the `pipeline_version`?

Reply:
Right, I still need to fix export!

Reply:
Should be okay now